v3/newrelic/config.go

```diff
@@ -827,6 +838,12 @@ func copyConfigReferenceFields(cfg Config) Config {
 			cp.Labels[key] = val
 		}
 	}
+	if nil != cfg.CustomAttributes {
```

Comment:
Please don't use "yoda" conditionals. "if cfg.CustomAttributes != nil" is the preferred way to state the expression.

Reply:
I know we have these in our legacy code, but we're trying to correct them as we find them and not put new ones in when writing new code :)